### PR TITLE
Handle deprecated providers with a error message and a URL to follow up

### PR DESCRIPTION
**Pull request by tonybaloney**
See - http://libcloud.apache.org/blog/2016/02/16/new-drivers-deprecated-drivers.html

### CODE REVIEW

**Comment by tonybaloney** [comment]
@Kami @erjohnso @jcsalterego @runseb @allardhoeve - this is an important one. Are you happy with the way this works?

**Comment by allardhoeve** [comment]
Looks simple enough to me.

**Comment by erjohnso** [comment]
LGTM @tonybaloney. Nice clean up!

**Comment by Kami** [comment]
Yay, removing code is always great (less stuff to maintainer, less surface area for bugs, etc.) :P

LGTM

File: libcloud/compute/providers.py [hunk 1]
```diff
@@ -190,6 +181,12 @@
 
 
 def get_driver(provider):
+    if provider in DEPRECATED_DRIVERS:
```
Comment (Kami) [inline libcloud/compute/providers.py:184]: A test case for this would be nice :)

File: libcloud/compute/providers.py [hunk 2]
```diff
@@ -190,6 +181,12 @@
...
+        url = DEPRECATED_DRIVERS[provider]['url']
+        reason = DEPRECATED_DRIVERS[provider]['reason']
+        msg = ('Provider no longer supported: %s, please visit: %s' %
+               (url, reason))
```
Comment (Kami) [inline libcloud/compute/providers.py:188]: Also, looks like it should be `% (reason, url)` - reason first.

**Pull request closed without merging**